### PR TITLE
Returning trading wallet id instead of client id and extended wallet DTO with api-key

### PR DESCRIPTION
**Pull request by polonzer**
https://lykkex.atlassian.net/browse/LWDEV-3833

Implemented:
returning trading wallet id instead of client id in 'GetBalances' and 'GetBalancesByAssetId'
getting related api-key for wallets in 'GetWallets', 'GetWallet', 'GetBalances' and 'GetBalancesByAssetId'

### CODE REVIEW

File: LykkeApi2/Controllers/WalletsController.cs [hunk 1]
```diff
@@ -23,6 +23,9 @@ namespace LykkeApi2.Controllers
     [Route("api/wallets")]
     public class WalletsController : Controller
     {
+        // todo: should be changed to Lykke.Service.ClientAccount.Client.AutorestClient.Models.WalletType.Trading after updating Lykke.Service.ClientAccount.Client nuget package
+        private const string TradingWalletType = "Trading";
```
Comment (tarurar) [inline LykkeApi2/Controllers/WalletsController.cs]: Please update package

File: LykkeApi2/Controllers/WalletsController.cs [hunk 2]
```diff
@@ -144,29 +163,21 @@ public WalletsController(IRequestContext requestContext, IClientAccountService c
...
-                Balances = clientBalances?.Select(ClientBalanceResponseModel.Create) ?? new ClientBalanceResponseModel[0]
-            });
 
             var wallets = await _clientAccountService.GetWalletsByClientIdAsync(_requestContext.ClientId);
-            if (wallets != null)
+            var clientKeys = await _hftInternalService.GetKeysAsync(_requestContext.ClientId);
+            foreach (var wallet in wallets)
             {
-                foreach (var wallet in wallets)
+                var balances = await _balancesClient.GetClientBalances(wallet.Type == TradingWalletType ? _requestContext.ClientId : wallet.Id);
```
Comment (tarurar) [inline LykkeApi2/Controllers/WalletsController.cs]: Need package update

File: LykkeApi2/Controllers/WalletsController.cs (hunk 1 above)
Comment (polonzer) [reply LykkeApi2/Controllers/WalletsController.cs]: Done.

File: LykkeApi2/Controllers/WalletsController.cs (hunk 2 above)
Comment (polonzer) [reply LykkeApi2/Controllers/WalletsController.cs]: Done.

**Pull request merged**